conda_smithy/ci_register.py

```diff
@@ -178,48 +178,38 @@ def add_project_to_travis(user, project):
     headers = travis_headers()
     endpoint = 'https://api.travis-ci.org'
 
-    url = '{}/hooks'.format(endpoint)
+    url = '{}/{}/{}/key'.format(endpoint, user, project)
```

Comment:
In the browser, yes it looks that way. However, if I run it locally (requiring JSON as we do), this is what I see.

```bash
$ curl -H "Accept: application/json" https://api.travis-ci.org/repos/conda-forge/foobar
{"file":"not found"}
```